### PR TITLE
Add QFMA/QFMS intrinsics

### PR DESCRIPTION
**Pull request by tlively**
Also re-enables the SIMD tests with broken cases commented out.

### CODE REVIEW

File: tests/test_wasm_builtin_simd.c [hunk 1]
```diff
@@ -957,26 +969,26 @@ int EMSCRIPTEN_KEEPALIVE __attribute__((__optnone__)) main(int argc, char** argv
...
   );
-  expect_vec(
-    i8x16_shl((i8x16){0, 1, 2, 4, 8, 16, 32, 64, -128, 3, 6, 12, 24, 48, 96, -64}, 8),
-    ((i8x16){0, 1, 2, 4, 8, 16, 32, 64, -128, 3, 6, 12, 24, 48, 96, -64})
-  );
+  /* expect_vec( */
+  /*   i8x16_shl((i8x16){0, 1, 2, 4, 8, 16, 32, 64, -128, 3, 6, 12, 24, 48, 96, -64}, 8), */
+  /*   ((i8x16){0, 1, 2, 4, 8, 16, 32, 64, -128, 3, 6, 12, 24, 48, 96, -64}) */
+  /* ); */
   expect_vec(
```
Comment (kripken) [inline tests/test_wasm_builtin_simd.c]: the description says "add", why were some tests commented out?

Comment (tlively) [reply tests/test_wasm_builtin_simd.c]: Sorry, I should have elaborated more. I will in the commit message. The SIMD tests were entirely disabled in #9241 due to a V8 bug with the shifts. The V8 bug has not been fixed yet, but I wanted the tests to run again, so I have commented out the failing tests.

Comment (kripken) [reply tests/test_wasm_builtin_simd.c]: Sounds good, thanks!

**Comment by tlively** [comment]
@kripken Something about the permissions must have changed because I can no longer re-run the failed jobs. Can you restart them for me please?

**Comment by kripken** [comment]
Restarted.

Can you not restart them even when logged in? If so we should file a ticket with them, if that's broken things are really going to get painful here... :(

**Comment by tlively** [comment]
Ah, sorry. I thought I was signed in but in fact I was not. Restarting permissions look normal now. Does this LGTY?

**Comment by kripken** [comment]
lgtm but I don't really follow the details :)

**Pull request merged**